vertical-pod-autoscaler/pkg/admission-controller/logic/recommendation_provider.go

```diff
@@ -61,17 +64,36 @@ func NewRecommendationProvider(vpaLister vpa_lister.VerticalPodAutoscalerLister,
 	}
 }
 
-func limitMatchesRequest(limit, request *resource.Quantity) bool {
-	// Limit not set, it cannot match the request.
-	if limit == nil || limit.Value() == 0 {
-		return false
+func limitMatchesRequest(originalLimit, originalRequest, recommendedRequest *resource.Quantity) *resource.Quantity {
```

Comment:
Function name does not match the contents any more.

Reply:
Done.